src/server/trpc/router/models/oauth.ts

```diff
@@ -1,6 +1,6 @@
 import { z } from "zod";
-import { router, publicProcedure } from "../../trpc";
-import { prisma } from "../../../db/client";
+import { router, publicProcedure, protectedProcedure } from "server/trpc/trpc";
+import { prisma } from "server/db/client";
 
 export const OAuthRouter = router({
   getOAuthProfileOwner: publicProcedure
```

Comment:
Not shown here but for lines 32-33, error catching should be defined instead of returning `null`, especially since the `findUniqueOrThrow` could be invoked quite a bit.

Reply:
These routes are unused as of now, will keep in mind convention when new routes are defined.
TRPC routes are generally suited for RPC fetching on page load, and fetching based off user interactions should be done via traditional `fetch` requests.